### PR TITLE
New version: Yubico.YubikeyManager version 1.2.6

### LINKED ISSUE #147714: [Update Request]: Yubico.YubikeyManager 1.2.6
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Yubico.YubikeyManager

### Package Version

1.2.6

### Please describe the changes you would like to see

There is a new release available, 1.2.6, addressing a security issue, [YSA-2024-01](https://www.yubico.com/support/security-advisories/ysa-2024-01/)

https://developers.yubico.com/yubikey-manager-qt/Releases/yubikey-manager-qt-1.2.6-win64.exe
https://developers.yubico.com/yubikey-manager-qt/Releases/yubikey-manager-qt-1.2.6-win32.exe

### PR DESCRIPTION
**Pull request by blowdart**
Resolves #147714

Checklist for Pull Requests
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Is there a linked Issue?

Manifests
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/147715)

### CODE REVIEW

File: manifests/y/Yubico/YubikeyManager/1.2.6/Yubico.YubikeyManager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  Scope: machine
+  InstallerUrl: https://developers.yubico.com/yubikey-manager-qt/Releases/yubikey-manager-qt-1.2.6-win64.exe
+  InstallerSha256: FC8F1C992C5B9F5D542549072CF58178A45AF31851561B22A768E35DBFD648B7
+- Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://developers.yubico.com/yubikey-manager-qt/Releases/yubikey-manager-qt-1.2.6-win32.exe
+  InstallerSha256: 4EB85788107C0AF89CA1ADE0779CAC9E86CF2D7A71C09B239FD478FF8ABD882C
+ManifestType: installer
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/y/Yubico/YubikeyManager/1.2.6/Yubico.YubikeyManager.installer.yaml:19]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/y/Yubico/YubikeyManager/1.2.6/Yubico.YubikeyManager.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Copyright: Copyright (c) Yubico 2020. All Rights Reserved.
+ShortDescription: Official tool to configure FIDO2, OTP and PIV functionality on your YubiKey
+Description: Official tool to configure FIDO2, OTP and PIV functionality on your YubiKey
+Moniker: yubimgr
+Tags:
+- 2fa
+- security
+- auth
+ManifestType: defaultLocale
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/y/Yubico/YubikeyManager/1.2.6/Yubico.YubikeyManager.locale.en-US.yaml:24]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/y/Yubico/YubikeyManager/1.2.6/Yubico.YubikeyManager.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Yubico.YubikeyManager
+PackageVersion: 1.2.6
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/y/Yubico/YubikeyManager/1.2.6/Yubico.YubikeyManager.yaml:7]:
```suggestion
ManifestVersion: 1.6.0
```

**Review by stephengillie — COMMENTED** [review]
Hi @blowdart,

While this manifest version is depreciated, most fields are identical with the latest manifest version. Usually, increasing the manifest version number is enough to pass this check. Also, reopening the PR should start the pipeline again.

**Comment by blowdart** [comment]
I'll try again then, because the command line tooling hates me 😂

**Pull request closed without merging**